internal/provider/resource_port_forward_test.go

```diff
@@ -0,0 +1,52 @@
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	//	"github.com/paultyng/go-unifi/unifi"
```

Comment:
```suggestion
```

Reply:
not sure what else I need, I want to add a bunch more test cases with multiport and leaving out fields, testing the auto-fill options. I'll change this file in the coming days probably